mmengine/optim/scheduler/param_scheduler.py

```diff
@@ -1146,10 +1149,10 @@ class CosineRestartParamScheduler(_ParamScheduler):
         restart_weights (list[float]): Restart weights at each
             restart iteration. Defaults to [1].
         eta_min (float, optional): Minimum parameter value at the end of
-            scheduling.
+            scheduling. Defaults to None.
         eta_min_ratio (float, optional): The ratio of minimum parameter value
             to the base parameter value. Either `eta_min` or `eta_min_ratio`
-            should be specified.
+            should be specified. Defaults to None.
```

Comment:
Update the sequence of the arguments in docstring too.

Reply:
it seems that there is no arguments update in `CosineRestartParamScheduler`